recipes/recipes_schubar.json

```diff
@@ -841,6 +841,30 @@
         "ice": "cubed",
         "prep": "build",
         "garnish": "Flag of lemon peel and raspberries"
+    },
+
+    "Cayuga White": {
+        "info": "Vintage 2013",
```

Comment:
You'll need to have an "ingredients" attribute, take a look a A Dram to see a similar type example. You can make a Red Wine and White Wine "recipe" and that should cover it all nicely. When a use looks at the drink, they'll see the list of what's available

Reply:
Thanks for the advice!